### PR TITLE
Feature/history page

### LINKED ISSUE #41: As a user, I can view a list of only articles I've previously read

### LINKED ISSUE #44: As a user, I can use the browser's forward and back buttons without losing my data
- [x] Implement local storage

### LINKED ISSUE #55: Shorten synopsis to prevent scrolling
Let's cut this down so the user doesn't have to scroll to see the explanation on mobile. 
```suggestion
        <p>Based on your mood and the articles you read, we dynamically sort your feed and nudge you toward sentimentally balance. We don't filter your feed.
        <p>If you prefer not to indicate your mood, press 'skip' and we'll sort based only on the articles you read.</p>
```

_Originally posted by @anthony-iacono in https://github.com/jphorner/wellnews/pull/54#pullrequestreview-799482675_

### PR DESCRIPTION
**Pull request by anthony-iacono**
### Features
- Closes #44 
- Closes #41
- Closes #55 

### Refactors 
- Moved sorting and history checking functionality from Feed up to App
- Removed back buttons in article view, now that data persists after use of browser buttons 
- Matched the sentiment scale across the app

### Issues and Technical Debt
- None

### Other Comments, Concerns, or Questions
- None

### Checklist
- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature work
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran Lighthouse and WAVE accessibility audits and fixed errors/warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by najinl — APPROVED** [review]

**Review by najinl — COMMENTED** [review]
We will need to remember to change the slice length to 9 or 10 before we make the final push, after updating tests! Great work!

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -24,18 +26,52 @@ const App = (): JSX.Element => {
...
             });
-
             setArticles(scoredArticles);
           });
       })
       .catch(error => setError(error.message));
   }, []);
 
+  useEffect((): void => {
+    console.log(userSentiment)
```
Comment (nzambonivergara) [inline src/components/App/App.tsx:40]: Remember to delete the console.logs :)

**Review by nzambonivergara — APPROVED** [review]
Great job implementing this, @anthony-iacono!!

File: src/components/App/App.tsx (hunk 1 above)
Comment (nzambonivergara) [inline src/components/App/App.tsx:40]:
```suggestion
```

File: src/components/App/App.tsx [hunk 2]
```diff
@@ -13,7 +14,8 @@ import History from '../History/History'
 
 const App = (): JSX.Element => {
   const [articles, setArticles] = useState<CleanedArticle[]>([]);
-  const [history, setHistory] = useState<string[]>([])
+  const [unreadArticles, setUnreadArticles] = useState<CleanedArticle[] | undefined>([])
```
[the remark this answers is not in the document]
Comment (anthony-iacono) [reply src/components/App/App.tsx:17]: Without it, I get this error. If you have any insight into a better way to resolve it, please share.
```
Failed to compile.

/Users/anthonyiacono/Sync/Turing/mod3/wellnews/src/components/App/App.tsx
TypeScript error in /Users/anthonyiacono/Sync/Turing/mod3/wellnews/src/components/App/App.tsx(46,23):
Argument of type 'CleanedArticle[] | undefined' is not assignable to parameter of type 'SetStateAction<CleanedArticle[]>'.
  Type 'undefined' is not assignable to type 'SetStateAction<CleanedArticle[]>'.  TS2345

    44 |   useEffect((): void => {

    45 |     const unreadArticles = getUnreadArticles();
  > 46 |     setUnreadArticles(unreadArticles);
       |                       ^
    47 |   }, [articles, history])
    48 |
    49 |   const getSortedArticles = (): CleanedArticle[] => {
```

**Pull request merged**